src/ncorr_commandLine.cpp

```diff
@@ -553,92 +424,105 @@ int main(int argc, char *argv[]) {
 	else{
 		config_DIC_analysis = DIC_analysis_config::REMOVE_BAD_POINTS;
 	}
+	std::vector<Image2D> imgs;
+	for (json::iterator it = inputFile.find("images")->begin(); it != inputFile.find("images")->end(); ++it) {
+  		image_names.push_back(it->get<string>());
+	}
+	//string directoryInUse = getDirPath(getImagesArray[1]);
+	//return 0;
+	for (int i = 1; i < image_names.size(); i++) {
 
-	//Output settings
-	isEulerian =
-		!(split(split(firstLevelSplit[5], "\n", false)[1], "\t")[1].compare(
-			"Lagrangian") == 0);
-
-	csvExport = split(split(firstLevelSplit[5], "\n", false)[2], "\t")[1];
-
-	videoExport = split(split(firstLevelSplit[5], "\n", false)[3], "\t")[1];
-
-	units = split(split(firstLevelSplit[5], "\n", false)[4], "\t")[1];
-
-	units_per_px = atof(
-		split(split(firstLevelSplit[5], "\n", false)[5], "\t")[1].c_str());
-
-	fps = atof(
-		split(split(firstLevelSplit[5], "\n", false)[6], "\t")[1].c_str());
-
-	openCV_color = split(split(firstLevelSplit[5], "\n", false)[7], "\t")[1];
-
-	end_delay = atof(
-		split(split(firstLevelSplit[5], "\n", false)[8], "\t")[1].c_str());
-
-	fourccInput = split(split(firstLevelSplit[5], "\n", false)[9], "\t")[1];
-
-	colorbar =
-		(split(split(firstLevelSplit[5], "\n", false)[10], "\t")[1].compare(
-			"true") == 0);
-
-	axes = (split(split(firstLevelSplit[5], "\n", false)[11], "\t")[1].compare(
-		"true") == 0);
-
-	scalebar =
-		(split(split(firstLevelSplit[5], "\n", false)[12], "\t")[1].compare(
-			"true") == 0);
-
-	numUnits = atof(
-		split(split(firstLevelSplit[5], "\n", false)[13], "\t")[1].c_str());
-
-	fontSize = atof(
-		split(split(firstLevelSplit[5], "\n", false)[14], "\t")[1].c_str());
-
-	tickMarks = atof(
-		split(split(firstLevelSplit[5], "\n", false)[15], "\t")[1].c_str());
+		imgs.push_back(image_names[i]);
+	}
+	string roiPath = inputFile.find("roi")->get<string>();
 
-	strainMin = atof(
-		split(split(firstLevelSplit[5], "\n", false)[16], "\t")[1].c_str());
+	DIC_input = DIC_analysis_input(imgs, 						// Images
+			ROI2D(Image2D(roiPath).get_gs() > 0.5),		// ROI
+			scaleFactor,                                      // scalefactor
+			interpType,			// Interpolation
+			subRegion,					// Subregion shape
+			radius,                                      // Subregion radius
+			numThreads,                                      // # of threads
+			config_DIC_analysis, // DIC configuration for reference image updates
+			false);	
 
-	strainMax = atof(
-		split(split(firstLevelSplit[5], "\n", false)[17], "\t")[1].c_str());
+}
 
-	dispMin = atof(
-		split(split(firstLevelSplit[5], "\n", false)[18], "\t")[1].c_str());
+int main(int argc, char *argv[]) {
+	
+	if (argc != 3) {
+		throw std::invalid_argument(
+			"Must have 2 command line input of either 'calculate' or 'load' and the path of the job file");
+	}
+	try{
+	// Initialize DIC and strain information ---------------//

```

Comment:
Indent on try-catch block missing. Any linters being used?

Reply:
Good point. Need to clean this up more. 4 am garbage.